tests/console/docker.pm

```diff
@@ -108,6 +108,15 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run supports --init parameter
+    record_info 'run --init', 'Test if docker run supports --init parameter';
+    assert_script_run 'docker run --rm --init opensuse/tumbleweed ps --no-headers -xo "pid args" | grep "1 /dev/init"';
+
+    # docker has all necessary man pages
+    record_info 'man pages', 'Check if docker has all necessary man pages';
+    assert_script_run 'man -P cat docker build | grep "docker-build - Build an image from a Dockerfile"';
```

Comment:
This will fail on every distribution without man pages (CaaSP, JeOS, ....) Here you can test that man is not present ;)

Reply:
I like this idea. Check if man is present or not and test or skip accordingly ;)